modules/osd-applications-renew-custom-domains.adoc

```diff
@@ -32,4 +32,7 @@ $ oc patch customdomain <company_name> --type='merge' -p '{"spec":{"certificate"
 [source,terminal]
 ----
 $ oc delete secret <secret-old> -n <my_project>
-----
\ No newline at end of file
+----
+
+.Troubleshooting
+* link:https://access.redhat.com/solutions/5419501[Error creating TLS secret]
```

Comment:
In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/applications/deployments/osd-config-custom-domains-applications.html#osd-applications-renew-custom-domains_osd-config-custom-domains-applications